ImplementationGuide/markdown/Binary/Binary_Motivation.md

```diff
@@ -6,4 +6,7 @@ Dazu werden die Daten base64-codiert in der Binary-Ressource (in XML oder JSON)
 Binary-Ressourcen werden von Attachment-Elementen verlinkt und damit in den Kontext anderer FHIR-Ressourcen
 (z.B. Patient und Encounter) gestellt.
 
+Im Rahmen des ISIK Basismoduls ist eine Implementierung des ISIKBinary-Profils nicht zwingend notwendig.
```

Comment:
Sorry, muss nochmal meckern ;-) Ich fürchte auch die Formulierung "nicht zwingend notwendig" ist hier falsch, oder zumindest irreführend. Das klingt so, als sollte man das eigentlich schon implementieren, muss aber nicht unbedingt. Tatsächlich ist es ja so, dass es in Basis überhaupt keinen UseCase für Binary gibt. Eine Implementierung wäre von daher komplett sinnfrei und sollte unbedingt unterlassen werden, es sei denn, man implementiert neben Basis auch noch Terminplanung oder Dokumentenaustausch.

Ich würde das in "nicht erforderlich" umformulieren.

Reply:
oder vielleicht so: "Das ISIK-Binary-Profil ist _nicht_ Bestandteil der Implementierung des ISIK Basismoduls"